libnmstate/netapplier.py

```diff
@@ -62,6 +62,7 @@ def apply(desired_state, verify_change=True, commit=True, rollback_timeout=60):
     desired_state = copy.deepcopy(desired_state)
     validator.validate(desired_state)
     validator.validate_capabilities(desired_state, netinfo.capabilities())
+    validator.validate_services(desired_state)
```

Comment:
The code should merge into `validator.validate_capabilities()` as that function is checking the OVS capability and this PR is expand that capability check.

Reply:
Done, thanks!